node_interop/lib/atomics.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020, Anatoly Pulyaevskiy. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+/// Bindings for the Atomics JavaScript obect. Since this object only has static
+/// methods, it's exposed as a module so that it can be used in the standard
+/// Dart style.
+@JS()
+library node_interop.atomics;
```

Comment:
Also, it looks like Atomics is a global JS object, so we need to add an export of atomics in `lib/node.dart` which is where all the global stuff is exported.